### PR TITLE
Added voice command to create Talon application context

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
`"talon create app context"` will generate a new folder in `apps` with a Talon and a python file. 

Example from visual studio code: 

`apps/visual_studio_code/visual_studio_code.talon`
```talon
app: visual_studio_code
-

```

`apps/visual_studio_code/visual_studio_code.py`
```py
from talon import Module, Context, actions

mod = Module()
ctx = Context()

mod.apps.visual_studio_code = r"""
os: windows
and app.exe: Code.exe
"""

ctx.matches = r"""
os: windows
app: visual_studio_code
"""

# @mod.action_class
# class Actions:
```

### CODE REVIEW

**Comment by nriley** [comment]
Would recommend you create platform-specific .py/.talon files by default (e.g., visual_studio_code_win.py/talon). Some of us use Talon on more than one platform…

**Comment by AndreasArvidsson** [comment]
Is that standard for community now? When I look in the applications folder it's quite rare.

**Comment by nriley** [comment]
It's a standard we discussed when we did our last rearrangement. Since keyboard commands usually differ between platforms, and my feeling is that if you're using this feature, you may not be an advanced user enough to be able to build a cross-platform implementation, so it's better for the defaults to be "safe".

**Comment by AndreasArvidsson** [comment]
Sounds reasonable

**Comment by AndreasArvidsson** [comment]
@nriley Better?

**Comment by nriley** [comment]
I'd recommend adding the platform suffix to the .talon file and the app: as well.

Also — if you're not watching the log, it doesn't seem like anything happens when I use this command. Perhaps consider opening the directory in the user's file browser, or at the least popping up a notification?

**Comment by AndreasArvidsson** [comment]
I don't think it makes sense to have a talon file per os for a single app. Maybe We can talk about this in tomorrow's meetup?

**Comment by AndreasArvidsson** [comment]
@nriley In today's meetup we decided to make the platform name an optional word in the spoken phrase. The current implementation just repeats the command I would prefer to have a list(`user.platform`) instead. What do people think?

File: plugin/talon_helpers/create_app_context.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+APPS_DIR = Path(__file__).parent.parent.parent / "apps"
+
+mod = Module()
+
+
+@mod.action_class
+class Actions:
+    def talon_create_app_context(platform_suffix: str = None):
+        """Create a new python context file for the current application"""
```
Comment (nriley) [inline plugin/talon_helpers/create_app_context.py:16]: Maybe include something about the directory here

**Review by nriley — CHANGES_REQUESTED** [review]

**Review by nriley — APPROVED** [review]

**Pull request merged**